book/07-git-tools/sections/submodules.asc

```diff
@@ -384,199 +384,199 @@ Ito ay nangangahulugan na walang lokal na tinatrabahuang branch (tulad ng ``mast
+Awtomatikong pagsasama sa scripts/setup.sh
+SALUNGATAN (nilalaman): Kasalungatan sa pag-merge sa scripts/setup.sh
+Nagtala ng preimage para sa 'scripts/setup.sh'
+Nabigo ang awtomatikong pag-merge; ayusin ang mga kasalungtan at pagkatapos i-commit ang resulta.
+Hindi ma-merge ang 'c75e92a2b3855c9e5b66f915308390d9db204aca' sa path ng submodule na 'DbConnector'
 ----
 
-You can go into the submodule directory and fix the conflict just as you normally would.
+Maaari kang magpunta sa submodule na direktoryo at ayusin ang salungatan tulad ng karaniwan mong ginagawa.
 
 [[_publishing_submodules]]
-===== Publishing Submodule Changes
+===== Paglalahatla sa mga Pagbabago ng Submodule
 
-Now we have some changes in our submodule directory.
-Some of these were brought in from upstream by our updates and others were made locally and aren't available to anyone else yet as we haven't pushed them yet.
+Ngayon ay mayroon ng tayo iilang mga pagbabago sa ating submodule na direktoryo.
+Ilan sa mga ito ay dinala mula sa upstream sa pamamagitan ng ating mga update at ang iba ay ginawa sa lokal at hindi pa magagamit ng sinuman dahil hindi pa natin na-push ang mga ito.
 
 [source,console]
 ----
 $ git diff
 Submodule DbConnector c87d55d..82d2ad3:
-  > Merge from origin/stable
-  > updated setup script
-  > unicode support
-  > remove unnecessary method
-  > add new option for conn pooling
+  > Nag-merge mula sa origin/stable
+  > ina-update ang setup script
+  > unicode na suporta
+  > tinanggal ang hindi kinakailangang pamamaraan
+  > nagdagdag ng bagong opsyon para sa conn pooling
 ----
 
-If we commit in the main project and push it up without pushing the submodule changes up as well, other people who try to check out our changes are going to be in trouble since they will have no way to get the submodule changes that are depended on.
-Those changes will only exist on our local copy.
+Kung tayo ay nag-commit sa pangunahing proyekto at pi-nush ito, nang wala ring pag-push sa mga pagbabago ng submodule, ang ibang tao na nagsusubok na mag-check out ng mga pagbabago ay magkakaroon ng problema dahil wala silang paraan upang makuha ang mga naka-dependeng pagbabago sa submodule.
+Ang mga pagbabago iyon ay umiiral lamang sa ating lokal na kopya.
 
-In order to make sure this doesn't happen, you can ask Git to check that all your submodules have been pushed properly before pushing the main project.
-The `git push` command takes the `--recurse-submodules` argument which can be set to either ``check'' or ``on-demand''.
-The ``check'' option will make `push` simply fail if any of the committed submodule changes haven't been pushed.
+Para makasiguro na hindi na ito mangyayari, maaari mong hilingin ang Git na suriin na ang lahat ng iyong mga submodile ay na-push nang maayos bago mag-push sa pangunahing proyekto.
+Ang utos `git push` ay tumatanggap ng argumento na `--recurse-submodules` na kung saan maaari itakda alinman sa ``check'' o ``on-demand''.
```

Comment:
"na kung saan maaari itakda alinman sa" to "na kung saan ay maaaring itakda ang alinman sa"